index.html

```diff
@@ -223,23 +240,25 @@ <h4>Skills</h4>
                 </ul>
             </div>
         </div>
-        <div class="get-started">
+        <div class="get-started" id="get-started">
             <img src="img/yellow-triangle-getting-started.svg" alt="" class="yellow-get-started">
+            <div class="yellow-get-started-desktop"></div>
             <img src="img/red-square-getting-started.svg" alt="" class="red-get-started">
+            <div class="img-geometry-7"></div>
             <h2>Get started</h2>
-            <article>
+            <article class="text-get-started">
                 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
             </article>
             <div class="form-text">
                 <p id="form-name" class="form-details">Your name</p>
                 <p id="form-email" class="form-details">Your email</p>
-                <p id="form-text" class="form-details">Enter your message here...</p>
+                <p id="form-text" class="form-details text-area-text">Enter your message here...</p>
             </div>
             <form action="https://formspree.io/f/xzboarvz" method="post">
                 <ul>
-                    <li><input type="text" required maxlength="30"></li>
-                    <li><input type="email" required maxlength="50"></li>
-                    <li><textarea name="textarea" cols="30" rows="10" required maxlength="500"></textarea></li>
+                    <li><input type="text" required maxlength="30" id="list-name"></li>
+                    <li><input type="email" required maxlength="50" id="list-email"></li>
+                    <li><textarea name="textarea" cols="30" rows="10" required maxlength="500" id="list-textarea"></textarea></li>
                     <li><button type="submit" class="btn-get-started">Start collaboration</button></li>
                 </ul>
```

Comment:
- [ ] Please add hover and pressed effects on buttons as provided in Figma: 
<img width="743" alt="Screenshot 2022-03-31 at 16 34 22" src="https://user-images.githubusercontent.com/77038610/161055773-8e00238f-9b97-430d-a304-9e932054659c.png">

Reply:
Added hover and pressed effects on buttons on both versions of the site.